apigw-lambda-translate-cdk/README.md

```diff
@@ -0,0 +1,89 @@
+# API Gateway -> Lambda -> Translate
+This patterns shows CDK deployment on how to leverage Amazon API Gateway, AWS Lambda, and Amazon Translate to perform language translation in a serverless fashion.
+
+## Architecture
+![Diagram](src/architecture.png)
+
+### What resources will be created?
```

Comment:
Please add a line that the API GW is not using authorization or authentication and should only be used for demo purposes

Reply:
This is done. Updated README.md with caution note.